src/dispatchers/touchDispatcher.ts

```diff
@@ -0,0 +1,138 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export type TouchCallback = (p: Point) => any;
+
+  export class Touch extends AbstractDispatcher {
```

Comment:
Possibly for documentation reasons, but we may want to document what this Dispatcher does (say what events does it listen for etc.).  That said, we don't have consistent documentation on the class level, so NBD if this doesn't get done.  Note that I'll probably think about this in the near future as we prepare for 1.0